cmd/stack.go

```diff
@@ -198,19 +198,25 @@ func setupStackCommand() *cobraext.Command {
 				return cobraext.FlagParsingError(err, cobraext.ProfileFlagName)
 			}
 
+			s, err := cmd.Flags().GetString(cobraext.ShellInitShellFlagName)
```

Comment:
Please use a more meaningful name here.
```suggestion
			shellName, err := cmd.Flags().GetString(cobraext.ShellInitShellFlagName)
```

Reply:
fixed in [`18364c6` (#921)](https://github.com/elastic/elastic-package/pull/921/commits/18364c69f1cb0aa809e48a32b48d538767b939c5)